django/contrib/admin/options.py

```diff
@@ -1728,6 +1728,22 @@ def changelist_view(self, request, extra_context=None):
             'admin/change_list.html'
         ], context)
 
+    def get_deleted_objects(self, objs, opts, request, using):
+        """
+        Hook for overriding the objects that are permitted to be deleted
+        or for customizing the format of the list of strings returned to the
+        template.
+
+        Find all objects related to ``objs`` that should also be deleted.
+        ``objs`` is a homogeneous iterable of objects (usually a QuerySet).
+
+        Return a 4-tuple containing deleteable objects (a nested list of
+        strings suitable for display in the template with the
+        ``unordered_list`` filter),the related object count, missing user
+        permissions and protected objects.
+        """
+        return get_deleted_objects(objs, opts, request.user, self.admin_site, using)
+
     @csrf_protect_m
     def delete_view(self, request, object_id, extra_context=None):
```

Comment:
I think a test for this change is missing. This would probably go in `admin_views` whereever the other tests for the `delete_view` are.

Reply:
I'm not sure I understand what this comment is referring to.  There is no change to the `delete_view` other than the return statement, which you commented on below.  Can you please clarify?